yext/answers-core
# Add support for Auth Token

- update AnswersConfig interface to a type that requires either apiKey or token (can provide both as well)
- update HttpService to add Authorization header with token
- update other services to pass token to httpService calls
- Add tests for auth token

J=SLAP-1626
TEST=auto & manual

- see that updated tests with token in config pass
- use local answers-core in SDK and see that token is in request authorization header
